src/components/core-player/CorePlayer.tsx

```diff
@@ -34,6 +34,10 @@ export interface CorePlayerProps {
 	/** `CorePlayer` initial state */
 	initialState?: CorePlayerInitialState;
 	children: ReactNode;
+	/** Time in sec for values that will set `onAlarmUpdate` */
+	timeAlarm?: number[];
```

Comment:
Could we improve naming to communicate that these are multiple times, e.g. `alarmTimes` (plural), `alarmSecs` or even just `alarms`?